.github/workflows/test_k6_cronjob.yml

```diff
@@ -49,6 +49,10 @@ jobs:
         if: always()
         uses: jwalton/gh-docker-logs@v2
 
+      - name: Print job status
```

Comment:
This part is left over testing part that came from my branch. Something probably went wrong with a rebase later. This part can be removed